spec/models/user_spec.rb

```diff
@@ -37,7 +37,7 @@
     end
   end
 
-  describe 'signed_in_before() method' do
+  describe 'signed_in_before' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.